### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Acorn-Studios**
Alphabetized and sorted the /usage category
and more

### CODE REVIEW

File: pages/components/_meta.json
Comment (asadm) [inline pages/components/_meta.json:1]: I wouldn't sort these actually. These are sorted by importance right now 🤔

File: pages/setup.mdx [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 ```js copy
 import { onPlayerJoin, insertCoin, isHost, myPlayer } from "playroomkit";
 ```
-
 ## Using a CDN
 
 You can also directly include it via a script tag from a CDN like `unpkg.com` or our own `cdn.joinplayroom.com`, make sure to also include the peer dependencies also (react, react-dom):
 
+
+### Using `cdn.joinplayroom.com`
```
Comment (asadm) [inline pages/setup.mdx:16]: we can discard this one, just unpkg is fine, (remove the "Using unpkg" heading)

**Review by asadm — CHANGES_REQUESTED** [review]
looks good, just a few changes needed.

File: pages/components/_meta.json
Comment (Acorn-Studios) [reply pages/components/_meta.json:1]: Yeah that makes sense.

**Review by Acorn-Studios — COMMENTED** [review]
Look over these and tell me what you think

**Pull request merged**